ui/src/speeddial.cpp

```diff
@@ -91,6 +91,7 @@ SpeedDial::SpeedDial(QWidget* parent)
     , m_tapTickTimer(NULL)
     , m_tapTick(false)
     , m_visibilityMask(DEFAULT_VISIBILITY_MASK)
+    , m_tapFeedbackType(Blink)
```

Comment:
Siegmund, move this line before the above one. We have enabled check that initialization here  must be in order of declaration in the .h file, since that's how it happens anyway (.cpp order doesn't matter, the check is to avoid false information from constructor). This will fix travis build.